xample-malware/mebromi.md

```diff
@@ -4,19 +4,27 @@
 |**Aliases**||
 |**Platforms**|Windows|
 |**Year**|2011|
-|**Associated ATT&CK Software**|None|
+|**Associated ATT&CK Software**|[Mebromi](https://attack.mitre.org/software/S0001/)|
 
 
 Mebromi
 =======
 A BIOS bootkit.
 
+
+ATT&CK Techniques
+-----------------
+|Name|Use|
+|---|---|
+|[Defense Evasion::Pre-OS Boot::System Firmware (T1542.001)](https://attack.mitre.org/techniques/T1542/001/)|Malware performs BIOS modification and can download and execute a file as well as protect itself from removal [[1]](#1)|
```

Comment:
When malware is in ATT&CK, only techniques that have been enhanced in MBC (or techniques that were missed in ATT&CK) should be in the list under "ATT&CK techniques". There should also be a statement like, "Please see [Mebromi](https://attack.mitre.org/software/S0001/) for a complete list of ATT&CK techniques" (maybe in the last line of the table?)

Reply:
Yup will do this next